doc/api/net.md

```diff
@@ -632,6 +632,8 @@ case of errors (parse error or so).
 If `exception` is specified, an [`'error'`][] event will be emitted and any
```

Comment:
Ouph, there isn't much consistency anywhere in net.md on how returns are documented. I'll update #13531 to fix the inconsistencies, and then update this one.